exe/scarpe

```diff
@@ -55,6 +55,13 @@ end
 use_dev = ARGV.delete("--dev") ? true : false
 use_debug = ARGV.delete("--debug") ? true : false
 
+if use_dev or use_debug
```

Comment:
I would probably do this on use_dev but *not* use_debug.

Use_dev is when you're doing it locally from a repo. But use_debug is perfectly valid even if you're using Shoes outside any cloned Scarpe repo, just in your app somewhere.

Reply:
I'm not sure how we're gonna do the Rails-style dir-full-of-app stuff, but right now a lot of Shoes apps don't *use* a Gemfile.